shell/platform/android/io/flutter/app/FlutterApplication.java

```diff
@@ -18,4 +19,12 @@ public void onCreate() {
         super.onCreate();
         FlutterMain.startInitialization(this);
     }
+
+    private Activity mCurrentActivity = null;
+    public Activity getCurrentActivity() {
```

Comment:
What is the intended use of getCurrentActivity?

Reply:
When the background service for the alarm manager comes up, it will try to use the FlutterNativeView from the current activity: https://github.com/flutter/plugins/pull/270/files#diff-1b20e736f0acfe196ade421bb4c33f8dR105